src/libs/antares/study/parts/hydro/prepro.cpp

```diff
@@ -145,11 +145,10 @@ bool PreproHydro::saveToFolder(const AreaName& areaID, const char* folder)
     return false;
 }
 
-bool PreproHydro::loadFromFolder(Study& s, const AreaName& areaID, const char* folder)
+bool PreproHydro::loadFromFolder(Study& s, const AreaName& areaID, const std::string& folder)
 {
     /* Asserts */
-    assert(folder);
-    assert('\0' != *folder);
+    assert(!folder.empty());
```

Comment:
Asserts only occur in debug. @flomnes Should we create a dedicated eror or exception for this ?

Reply:
No need, this check is irrelevant. If `folder` is empty, we'll have an error later (file/directory does not exist, probably).